### ISSUE TITLE
androidx.core version 1.7.0-alpha02 dependency causing Android build errors

### ISSUE THREAD

**Issue by stephanie-finch** [luanpotter/audioplayers#999]
The audioplayers' library dependency on `androidx.core:core-ktx:+` is using androidx.core:core-ktx:1.70-alpha2 (requires compileSdk 31) even though our compileSdk is 29.

**Full Description**
Without any new local changes or library updates, we started encountering Android build errors when trying to run the emulator: `error: resource android:attr/lStar not found`. From investigation, we saw others encounter issues if they had third party libraries not specify a safe core-ktx version: https://stackoverflow.com/a/69031863.

We found the androidx.core:core-ktx dependency in the audioplayers library by running `gradle :app:dependencies > dependencies.txt` in the command line and searching `core-ktx` in the .txt file:
```
+--- project :audioplayers
|    +--- io.flutter:flutter_embedding_debug:1.0.0-05e680e202af9a92461070cb2d9982acad46c83c (*)
|    +--- androidx.core:core-ktx:+ -> 1.7.0-alpha02
```

**Code to Reproduce**
`audioplayers: ^0.18.3` in pubspec.yaml.

**Log Errors**
```
error: resource android:attr/lStar not found
```

**Files/URLs/Sources**
Related links:
- https://community.revenuecat.com/sdks-51/androidx-core-version-1-7-0-alpha02-dependency-causing-android-build-errors-397?postid=1025#post1025
- https://issuetracker.google.com/issues/199180389
- https://stackoverflow.com/a/69031863

**Platforms**
Pixel 2 XL API 30 (android-x86 emulator)
Pixel 3a API 30 (android-x86 emulator)

**Workaround**
Since our Android compileSdk is 29, our workaround was to add the following to the bottom of `app/build.gradle`:
```
configurations.all {
  resolutionStrategy {
    force 'androidx.core:core:1.6.0'
    force 'androidx.core:core-ktx:1.6.0'
  }
}
```

**Comment by luanpotter** [comment]
Wasn't this [fixed already](https://github.com/luanpotter/audioplayers/pull/992)?

Could you please try with the current version `0.20.0`?

**Comment by troyshu** [comment]
I was getting the same error as OP, updating `audioplayers` to `0.20.0` in my pubspec.yaml worked for me. Thank you!

**Comment by comESK** [comment]
`0.20.0` also fixed for me, but then I also lost Android audio like in #1001

**Comment by luanpotter** [comment]
Awesome! I will fix #1001 right away, that is a much simpler issue.
